core/boot/src/test/java/org/eclipse/dataspaceconnector/boot/system/DefaultServiceExtensionContextTest.java

```diff
@@ -56,7 +57,7 @@
     void setUp() {
         TypeManager typeManager = new TypeManager();
         Monitor monitor = mock(Monitor.class);
-        Telemetry telemetry = mock(Telemetry.class);
+        Telemetry telemetry = new Telemetry(OpenTelemetry.noop());
```

Comment:
There are other parts in the code where we mocked Telemetry. Perhaps you can adapt those too.